SpywareFilter/sections/general_url.txt

```diff
@@ -98,6 +98,7 @@ directtrack.*.js
 /watch.*.js?key=*&kw=*&refer=
 /pv_count.
 _analytics.js?
+/static/js/analytics-helper.js
```

Comment:
I don't see this request, btw it doesn't look popular, so it's better to add to specific with $domain modifier
(or remove this rule if this request does not exist anymore)

Reply:
Let's take risk. The tracking protection filter is so outdated

Reply:
EasyPrivacy beats it

Reply:
@Alex-302 please decide

Reply:
https://www.ndtv.com/static/js/analytics-helper.js

Reply:
But it is just google analytics, low priority issue.